### PR TITLE
Cleanup moveit_group_interface tutorial:

### PR DESCRIPTION
**Pull request by davetcoleman**
- Created constant for planning group name
- Formatted whole file
- Renamed "group" var to "move_group" to differentiate from joint_model_group
- Added improved explanations
- Reduced complex lines

Students in my lab are learning MoveIt! now: https://github.com/ros-planning/moveit_tutorials/issues/37

### CODE REVIEW

File: doc/pr2_tutorials/planning/src/move_group_interface_tutorial.cpp [hunk 1]
```diff
@@ -50,36 +50,40 @@ int main(int argc, char **argv)
...
   // Getting Basic Information
   // ^^^^^^^^^^^^^^^^^^^^^^^^^
   //
   // We can print the name of the reference frame for this robot.
-  ROS_INFO("Reference frame: %s", group.getPlanningFrame().c_str());
+  ROS_INFO("Reference frame: %s", move_group.getPlanningFrame().c_str());
 
   // We can also print the name of the end-effector link for this group.
-  ROS_INFO("Reference frame: %s", group.getEndEffectorLink().c_str());
+  ROS_INFO("Reference frame: %s", move_group.getEndEffectorLink().c_str());
```
Comment (v4hn) [inline doc/pr2_tutorials/planning/src/move_group_interface_tutorial.cpp]: I suppose, this should actually be "Endeffector" instead of "Reference frame". (I know this is no mistake in your patch...)

File: doc/pr2_tutorials/planning/src/move_group_interface_tutorial.cpp [hunk 2]
```diff
@@ -140,17 +140,21 @@ int main(int argc, char **argv)
...
+  std::vector<double> joint_group_positions;
+  current_state->copyJointGroupPositions(joint_model_group, joint_group_positions);
 
-  // Now, let's modify one of the joints, plan to the new joint
-  // space goal and visualize the plan.
-  group_variable_values[0] = -1.0;
-  group.setJointValueTarget(group_variable_values);
-  success = group.plan(my_plan);
+  // Now, let's modify one of the joints, plan to the new joint space goal and visualize the plan.
+  joint_group_positions[0] = -1.0;
```
Comment (v4hn) [inline doc/pr2_tutorials/planning/src/move_group_interface_tutorial.cpp]: Would it make sense to specify something like `-M_PI/4`, so that people realize the joint position is radian in this case? -1 is not your typical radian value.. and I met way too many students who didn't understand radian at all...

File: doc/pr2_tutorials/planning/src/move_group_interface_tutorial.cpp [hunk 3]
```diff
@@ -224,25 +227,27 @@ int main(int argc, char **argv)
 
   // We want the cartesian path to be interpolated at a resolution of 1 cm
   // which is why we will specify 0.01 as the max step in cartesian
-  // translation.  We will specify the jump threshold as 0.0, effectively
-  // disabling it.
+  // translation.  We will specify the jump threshold as 0.0, effectively disabling it.
```
Comment (v4hn) [inline doc/pr2_tutorials/planning/src/move_group_interface_tutorial.cpp]: This is not really related to this patch, but should the `jump_threshold` really be disabled?
This is relevant to safety and I feel teaching people to disable it is not a good idea..

File: doc/pr2_tutorials/planning/src/move_group_interface_tutorial.cpp (hunk 1 above)
Comment (davetcoleman) [inline doc/pr2_tutorials/planning/src/move_group_interface_tutorial.cpp]: fixed

File: doc/pr2_tutorials/planning/src/move_group_interface_tutorial.cpp (hunk 3 above)
Comment (davetcoleman) [inline doc/pr2_tutorials/planning/src/move_group_interface_tutorial.cpp]: I agree with you - i would never disable the jump threshold. but i don't want to get into that here. ive added a comment on safety

**Comment by v4hn** [comment]
makes sense to me.
I approve.
